tests/interp/abort-hot-comment/test.ngc

```diff
@@ -2,7 +2,7 @@
 #<named> = 4711
 o100 while [#42 lt 100]
 o200 if [#42 eq 20]
-     (abort, MixedCase param42=#42 named=#<named> ini=#<_ini[a]value>)
+     (abort, MixedCase param42=#42 named=#<named> INI=#<_ini[a]value>)
```

Comment:
same here

Reply:
I think this is correct IMHO since it refers to the value that is parsed from the INI file.